### ISSUE TITLE
What are all hostnames that CLI tool reaches out to on the internet?

### ISSUE THREAD

**Issue by alan-czajkowski** [dependency-check/DependencyCheck#6600]
I have internet completely blocked at my organization, but I am able to whitelist certain hostnames.

What are all hostnames that CLI tool reaches out to on the internet?

**Comment by alan-czajkowski** [comment]
@jeremylong is this information documented somewhere?

**Comment by jeremylong** [comment]
I don't think it is "documented" anywhere but the source code. Luckily - just look for URLs in https://github.com/jeremylong/DependencyCheck/blob/main/core/src/main/resources/dependencycheck.properties

There aren't that many.

**Comment by OrangeDog** [comment]
The actual `nvd.api.datafeed.url` is not currently in that file.

**Comment by jeremylong** [comment]
The `nvd.api.datafeed.url` isn't used unless you specifically set it.

**Comment by OrangeDog** [comment]
But the tool still talks to the API at the default location, which isn't listed in that file.

**Comment by SachinAditya** [comment]
Hi, I’d like to work on documenting the hostnames used by the CLI. If no one is currently working on this, may I take it?

**Comment by chadlwilson** [comment]
> Hi, I’d like to work on documenting the hostnames used by the CLI. If no one is currently working on this, may I take it?

Sure.

Probably better to refactor / tidy up https://dependency-check.github.io/DependencyCheck/data/index.html ([source here](https://github.com/dependency-check/DependencyCheck/blob/main/src/site/markdown/data/index.md)) and document it in a way that is independent of use of the CLI vs various plugins. All ODC variants essentially have the same (possible) data sources depending on configuration options.

**Comment by SachinAditya** [comment]
Thanks! I’ll work on refactoring and improving the data sources documentation to cover all variants (CLI and plugins). I’ll open a PR once ready.
